integration/commands_diagnostic_test.go

```diff
@@ -125,6 +125,28 @@ func TestCommandsDiagnosticGetLog(t *testing.T) {
 	}
 }
 
+func TestCommandsDiagnosticGetLogTelemetryReported(t *testing.T) {
+	t.Parallel()
+	res := setup.SetupWithOpts(t, &setup.SetupOpts{
+		DatabaseName: "admin",
+	})
+
+	ctx, collection := res.Ctx, res.Collection
+
+	command := bson.D{{"getLog", "startupWarnings"}}
+
+	var actual bson.D
+	err := collection.Database().RunCommand(ctx, command).Decode(&actual)
+	require.NoError(t, err)
+
+	m := actual.Map()
+	log := m["log"].(bson.A)
+	assert.Equal(t, 3, len(log))
+
+	// check that the last line contains info about telemetry being undecided
+	assert.Contains(t, log[2].(string), "Telemetry state undecided")
```

Comment:
That test assumes that telemetry state is undecided, but nothing actually ensures that. I think we should check it in common setup

Reply:
Fixed